dislash/slash_commands/slash_client.py

```diff
@@ -1035,7 +1059,11 @@ async def wait_for_button_click(self, check=None, timeout=None):
 
     async def wait_for_dropdown(self, check=None, timeout=None):
         if check is None:
-            check = lambda ctx: True
+            def check(inter):
+                if message.id != inter.message.id:
+                    return False
+                else:
+                    return True

```

Comment:
```suggestion
check = lambda ctx: True
```
Simmilar to [this](https://github.com/EQUENOS/dislash.py/pull/35/files#r671050405)

Reply:
but pep-8 say do not assign a lambda expression, use a def